util/ipresolver/__init__.py

```diff
@@ -196,3 +206,6 @@ def _parse_amazon_ranges(ranges):
                 all_amazon[region].add(IPNetwork(service_description["ip_prefix"]))
 
         return all_amazon
+
+    def get_ip_region(self, ip):
```

Comment:
Not implemented, and I don't see it being called anywhere.

Reply:
Removed